README.md

```diff
@@ -1 +1,50 @@
 # Bigcommerce Sample App: Hello World in Ruby and Sinatra
+
+## App Registration
+ - Create a trial shop on [Bigcommerce](https://www.bigcommerce.com/)
+ - Go to the [developer portal](https://developer.bigcommerce.com/) and log in
+ - Go to "My Apps" (linked in the navbar)
+ - Click the button "Create an app", enter a name for the new app and then click the button "Create"
+ - Go right to section 4 (Technical) in the app dialog and enter the following URLs:
+   - *Auth Callback URL* - `https://<app hostname>/auth/bigcommerce/callback`
+   - *Load Callback URL* - `https://<app hostname>/load`
+   - *Uninstall Callback URL* - `https://<app hostname>/uninstall`
+ - Enable the following permission scopes under *OAuth scopes*:
+   - *Orders* - MODIFY
+   - *Products* - MODIFY
+   - *Customers* - MODIFY
+   - *Content* - MODIFY
+   - *Marketing* - MODIFY
+   - *Information* - READ-ONLY
+ - Finally, click `Save & Close` on the top right of the dialog
+
+## App Setup
+
+### Heroku
+
+*Note: it is assumed that one already has a Heroku account, has the Heroku toolbelt installed, and has authenticated with the toolbelt*
```

Comment:
Please add a comment here that this is just one way to deploy your app, not the only way.